tesla_ble_mqtt/translations/fr.yaml

```diff
@@ -20,6 +20,9 @@ configuration:
   debug:
     name: Journalisation du débogage
     description: Imprimer des messages détaillés dans le journal pour le débogage
+  temperature_unit_fahrenheit:
+    name: Unité de température; Fahrenheit si vrai sinon Celsius
```

Comment:
Same as english version

Reply:
Any country in that world that speak French and uses Fahrenheit? :D

Reply:
To answer that question, Liberia and USA.